kimchi/src/tests/range_check.rs

```diff
@@ -544,8 +544,8 @@ fn verify_range_check0_test_copy_constraints() {
                     typ: cs.gates[row].typ,
                     src: Wire::create(row as i32, col as i32),
                     dst: Wire::create(
-                        row as i32 + 3 - row as i32,
-                        2 * (row as i32) + col as i32 + 2
+                        (row as i32) + 3 - (row as i32),
```

Comment:
We should not use `Wire::create` but I've fixed that in another PR, which is smaller and will probably get merged first.